src/app/feed/feed-header/feed-header.component.scss

```diff
@@ -10,7 +10,7 @@
 		flex-wrap: wrap;
 		align-items: center;
 		width: 100%;
-		height: 85px;
+		height: 155px;
```

Comment:
I was talking about the height which is there above the search bar and the logo.

![screenshot 97](https://cloud.githubusercontent.com/assets/11540785/24954865/b728ef52-1f9e-11e7-8052-eafab7217fd1.png)